README.md

```diff
@@ -9,30 +9,40 @@
 
 > Note: Looking for collaborators. [Want to help improve ts-jest?](https://github.com/kulshekhar/ts-jest/issues/223)
 
+`ts-jest` is a TypeScript preprocessor for Jest with source map support and 
+sensible defaults that quickly allows you to start testing TypeScript with Jest.
 
```

Comment:
Should we use 'defaults' instead of 'sensible defaults' here and in other places? We don't gain much by calling the defaults sensible. There will be a group, no matter how small, that won't find these defaults sensible :)